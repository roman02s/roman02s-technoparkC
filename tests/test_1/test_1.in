1

diet_1
123
24
12
50

1
2
A
0.123
2

2
1
123
