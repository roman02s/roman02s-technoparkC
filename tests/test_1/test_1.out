Введите действие:
 1 - ввести рацион
 2 - подобрать рацион
Введите название рациона питания
Введите энергетическую ценность
Введите содержание белков
Введите содержание жиров
Введите содержание углеводов
Есть добавки?
1 - Да
2 - Нет
Тип добавки:
1 - пищевые добавки
2 - витамины
3 - минералы
Введите название витамина
Введите кол-во
Есть добавки?
1 - Да
2 - Нет
Введите действие:
 1 - ввести рацион
 2 - подобрать рацион
Выберите критерии для поиска рациона:
1 - энергетическая ценность
2 - кол-во белков
3 - кол-во жиров
4 - кол-во углеводов
5 - наличие пищевых добавок
6 - наличие витаминов
7 - наличие минералов
Введите значение:
Подходящие варианты:
diet_1
